subprojects/docs/src/samples/userguide/buildlifecycle/projectEvaluateEvents/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+// tag::after-evaluate[]
+allprojects {
+    afterEvaluate {
+        if (property("hasTests") as Boolean) {
```

Comment:
:x: should be `extra["hasTests"]`, sorry for missing that one in my previous review

Reply:
Done @eskatos